cmd/argo/lint/formatter_pretty_test.go

```diff
@@ -1,3 +1,4 @@
+// disabled on windows because colorized output is not supported - go-color will return just the text without colors.
```

Comment:
I might be mistaken, but I thought this file tests the output messages themselves too, which include colors. the output message part would still be relevant, no?

Reply:
fixed, thanks for bringing this to my attention again :)